### PR TITLE
Don't query chain in Signer, use Redux isTest

### PR DESCRIPTION
**Pull request by jacogr**
No functionality changed, but allows me to break up the Signer changes into smaller chunks. 

This one (1 of n) gets rids of the uneeded queries for chain in the Signer. We have the information available on Redux, so no need to make extra queries for each component needing the information. As a bonus the `chain === 'testnet' || chain === 'morden'` change is now only done in one place in the UI, so it cleans up the codebase.

In short: don't query for chain per component and pass it along the chain, use the global state that indicates whether we are on the testnet or not.

Closes https://github.com/ethcore/parity/issues/3522

### CODE REVIEW

**Comment by ngotchac** [comment]
Missed a few places for `morden`:

* `src/dapps/basiccoin/services.js` L108
* `src/redux/providers/statusReducer.js` L34 (might need to change to `tesnet` whether than `morden` or `ropsten`)
* `src/views/Signer/components/Account/AccountLink/AccountLink.js` L60
* `src/views/Signer/components/TxHashLink/TxHashLink.js` L35

**Comment by ngotchac** [comment]
Might want to change `src/dapps/basiccoin/services.js` L108 and `src/redux/providers/statusReducer.js` L34 from `morden` to `testnet`
